lib/plugins/aws/custom-resources/resources/api-gateway-cloud-watch-role/handler.js

```diff
@@ -2,20 +2,15 @@
 
 const { wait, MAX_AWS_REQUEST_TRY } = require('../utils');
 const { getEnvironment, handlerWrapper } = require('../utils');
+const { GetAccountCommand, UpdateAccountCommand } = require('@aws-sdk/client-api-gateway');
 const {
-  APIGatewayClient,
-  GetAccountCommand,
-  UpdateAccountCommand,
-} = require('@aws-sdk/client-api-gateway');
-const {
-  IAMClient,
   ListAttachedRolePoliciesCommand,
   CreateRoleCommand,
   AttachRolePolicyCommand,
 } = require('@aws-sdk/client-iam');
+const AWSClientFactory = require('../../../../../aws/client-factory');
```

Comment:
Aren't these files the custom resource handlers?

If yes, they shouldn't be impacted by the changes in the CLI. And they wouldn't be able to require files from the CLI package (because they would run in Lambda).

Reply:
Ah, this was a fix to make sure they are re-using the same client.